crates/cli/src/analyze.rs

```diff
@@ -13,14 +16,23 @@ pub struct AnalyzeCommand {
     #[structopt(short = "i", long = "input-format", default_value = "json")]
     input_format: Format,
 
+    /// Path to the file that will be read from, or '-' to indicate stdin (default).
+    #[structopt(short = "f", default_value = "-")]
+    input_file: String,
```

Comment:
Can we do `input_file: Option<String>` and have `None` be `stdin`? This way we don't have to invent syntax for stdin (which I never liked about `clif-util`, it meant you always had to remember to check for `"-"` manually rather than letting the type system guide you) and we can support files named `-` (even if anyone actually making such a file is a terrible idea).

Reply:
(Same for the other subcommand and `output_file`)